css-view-transitions-2/Overview.bs

```diff
@@ -405,44 +416,57 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 		Prepend a step at the beginning of the task [=queue a global task|queued=] on |navigable|'s [=active window=]
 		when <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#apply-the-history-step">applying the history step</a> (14.11.1, <a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#updating-the-traversable:queue-a-global-task-3">here</a>):
 
-		If |changingNavigationContinuation| update-only is false, then [=setup outbound cross-document view transition=] given |oldDocument|, |newDocument| and the remaining steps and return from these steps.
+		If |changingNavigationContinuation| update-only is false, then [=setup cross-document view-transition=] given |oldDocument|, |newDocument|, |navigationType|, and the remaining steps and return from these steps.
 
 		Note: This would wait until a transition is captured or skipped before proceeding to unloading the old document and activating the new one.
 	</div>
 
 	<div algorithm="monkey patch to rendering">
 		Run the following step in [=update the rendering|updating the renedering=], before [=running the animation frame callbacks=]:
 
-		1. For each [=fully active=] {{Document}} |doc| in |docs|, run the [=ready to render=] steps for |doc|.
+		1. For each [=fully active=] {{Document}} |doc| in |docs|, run the [=reveal=] steps for |doc|.
 	</div>
 
 	<div algorithm="monkey patch to reactivation">
 		Run the following step at the end of [=Document/reactivate=]:
 
-		1. Set |document|'s [=ready to render fired=] to false.
+		1. Set |document|'s [=pagereveal fired=] to false.
 	</div>
 
-	<div algorithm="page ready to render">
-	When {{Document}} |document| is <dfn>ready to render</dfn>:
-		1. If |document|'s [=document/ready to render fired=] is true, then return.
+	<div algorithm="page reveal">
+		To <dfn>reveal</dfn> {{Document}} |document|:
+			1. If |document|'s [=document/page pagereveal fired=] is false, then:
+				1. Let |transition| be the result of [=activating cross-document view-transition=] for |document|.
 
-		1. Let |transition| be the result of getting the [=inbound cross-document view-transition=] for |document|.
+				1. [=Fire an event=] named <code>pagereveal</code> at |document|'s [=relevant global object=],
+					using {{PageRevealEvent}}, with [=PageRevealEvent/view transition=] initialized to .
```

Comment:
I think you meant "initialized to the [=inbound cross-document view-transition=]".

Reply:
Oops, should be |transition|. Will follow up.